integration/findandmodify_test.go

```diff
@@ -0,0 +1,357 @@
+// Copyright 2021 FerretDB Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package integration
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+	"go.mongodb.org/mongo-driver/bson"
+	"go.mongodb.org/mongo-driver/bson/primitive"
+	"go.mongodb.org/mongo-driver/mongo"
+
+	"github.com/FerretDB/FerretDB/integration/shareddata"
+)
+
+func TestFindAndModifySimple(t *testing.T) {
+	ctx, collection := setup(t, shareddata.Scalars, shareddata.Composites)
+
+	for name, tc := range map[string]struct {
+		command  any
+		response bson.D
+	}{
+		"EmptyQueryRemove": {
+			command: bson.D{
+				{"findAndModify", collection.Name()},
+				{"query", bson.D{}},
+				{"remove", true},
+			},
+			response: bson.D{
+				{"lastErrorObject", bson.D{{"n", int32(1)}}},
+				{
+					"value",
```

Comment:
can this doc representation be somehow similar to the one on line 57?

Reply:
I get it right you want to extract something from response. Looks like it's not an option as there would be difference in `lastErrorObject` between those 2 tests.